<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>817</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 31, 49);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>7</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>22</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>222</red>
         <green>221</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>222</red>
         <green>221</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>222</red>
         <green>221</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>222</red>
         <green>221</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>222</red>
         <green>221</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>222</red>
         <green>221</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>222</red>
         <green>221</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="Open_Image"/>
    <addaction name="Open_PDF"/>
    <addaction name="Save_file"/>
    <addaction name="separator"/>
    <addaction name="Save_project"/>
    <addaction name="Load_project"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="Undo"/>
    <addaction name="Redo"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="Grey_filter"/>
    <addaction name="Brightness_control"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="Author"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuEdit"/>
   <addaction name="menuImage"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Zoom_In_Out"/>
   <addaction name="Add_Text"/>
   <addaction name="Rotate_right"/>
   <addaction name="Rotate_left"/>
   <addaction name="Custom_rotation"/>
   <addaction name="Flip_horizontally"/>
   <addaction name="Flip_vertically"/>
   <addaction name="Resize_image"/>
   <addaction name="Crop"/>
   <addaction name="Excel_input"/>
   <addaction name="Impose_image"/>
   <addaction name="Print"/>
  </widget>
  <action name="actionOpen_image">
   <property name="text">
    <string>Open image</string>
   </property>
  </action>
  <action name="actionOpen_PDF">
   <property name="text">
    <string>Open PDF</string>
   </property>
  </action>
  <action name="Open_Image">
   <property name="text">
    <string>Open image</string>
   </property>
  </action>
  <action name="Open_PDF">
   <property name="text">
    <string>Open PDF</string>
   </property>
  </action>
  <action name="Undo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="statusTip">
    <string>Ctrl+Z</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="Save_file">
   <property name="text">
    <string>Save image</string>
   </property>
  </action>
  <action name="Save_One_Pdf">
   <property name="text">
    <string>Save single PDF</string>
   </property>
  </action>
  <action name="Print">
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/icons/icons8-print-80.png</normaloff>:/icons/icons8-print-80.png</iconset>
   </property>
   <property name="text">
    <string>File to print</string>
   </property>
  </action>
  <action name="Add_Text">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/icons/icons8-text-box-80.png</normaloff>:/icons/icons8-text-box-80.png</iconset>
   </property>
   <property name="text">
    <string>Add Text</string>
   </property>
  </action>
  <action name="Zoom_In">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/zoom_in.png</normaloff>:/icons/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom_In</string>
   </property>
  </action>
  <action name="Zoom_Out">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/zoom_out.png</normaloff>:/icons/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom_Out</string>
   </property>
  </action>
  <action name="Zoom_In_Out">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/icons/icons8-zoom-in-80.png</normaloff>:/icons/icons8-zoom-in-80.png</iconset>
   </property>
   <property name="text">
    <string>Zoom_In_Out</string>
   </property>
  </action>
  <action name="Edit_font">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edit font</string>
   </property>
  </action>
  <action name="Rotate_right">
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/icons/icons8-rotate-right-80.png</normaloff>:/icons/icons8-rotate-right-80.png</iconset>
   </property>
   <property name="text">
    <string>Rotate canvas right</string>
   </property>
  </action>
  <action name="Rotate_left">
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/icons/icons8-rotate-left-80.png</normaloff>:/icons/icons8-rotate-left-80.png</iconset>
   </property>
   <property name="text">
    <string>Rotate canvas left</string>
   </property>
  </action>
  <action name="Flip_horizontally">
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/icons/icons8-flip-horizontal-80.png</normaloff>:/icons/icons8-flip-horizontal-80.png</iconset>
   </property>
   <property name="text">
    <string>Flip horizontally</string>
   </property>
  </action>
  <action name="Flip_vertically">
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/icons/icons8-flip-vertical-80.png</normaloff>:/icons/icons8-flip-vertical-80.png</iconset>
   </property>
   <property name="text">
    <string>Flip vertically</string>
   </property>
  </action>
  <action name="Save_project">
   <property name="text">
    <string>Save project</string>
   </property>
  </action>
  <action name="Load_project">
   <property name="text">
    <string>Load project</string>
   </property>
  </action>
  <action name="Grey_filter">
   <property name="text">
    <string>Grey filter</string>
   </property>
  </action>
  <action name="Add_blur">
   <property name="text">
    <string>Add blur</string>
   </property>
  </action>
  <action name="Brightness_control">
   <property name="text">
    <string>Color and blur</string>
   </property>
  </action>
  <action name="Custom_rotation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/newPrefix/icons8-circled-notch-80.png</normaloff>:/newPrefix/icons8-circled-notch-80.png</iconset>
   </property>
   <property name="text">
    <string>Custom Rotation</string>
   </property>
  </action>
  <action name="Excel_input">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/newPrefix/icons8-select-all-80.png</normaloff>:/newPrefix/icons8-select-all-80.png</iconset>
   </property>
   <property name="text">
    <string>Excel input</string>
   </property>
  </action>
  <action name="Impose_image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/newPrefix/icons8-xlarge-icons-80.png</normaloff>:/newPrefix/icons8-xlarge-icons-80.png</iconset>
   </property>
   <property name="text">
    <string>Impose image</string>
   </property>
  </action>
  <action name="actionResize_canvas">
   <property name="text">
    <string>Resize image</string>
   </property>
  </action>
  <action name="Resize_image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/icons/icons8-resize-40.png</normaloff>:/icons/icons8-resize-40.png</iconset>
   </property>
   <property name="text">
    <string>Resize image</string>
   </property>
  </action>
  <action name="Crop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/newPrefix/icons8-crop-80.png</normaloff>:/newPrefix/icons8-crop-80.png</iconset>
   </property>
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="Redo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="statusTip">
    <string>Alt+Z</string>
   </property>
   <property name="shortcut">
    <string>Alt+Z</string>
   </property>
  </action>
  <action name="Author">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons/new.qrc"/>
 </resources>
 <connections/>
</ui>
